Userland/Services/WindowServer/Compositor.cpp

```diff
@@ -303,25 +305,21 @@ void Compositor::compose()
         check_restore_cursor_back(cursor_screen, cursor_rect);
 
     auto paint_wallpaper = [&](Screen& screen, Gfx::Painter& painter, Gfx::IntRect const& rect, Gfx::IntRect const& screen_rect) {
-        // FIXME: If the wallpaper is opaque and covers the whole rect, no need to fill with color!
```

Comment:
This FIXME actually still applies to the `WallpaperMode::Center` branch below. It'd be better to move it there instead of deleting it.

Reply:
Oh right, I have fixed it.